modules/home-manager/neovim.nix

```diff
@@ -5,11 +5,20 @@
   ...
 }:
 let
-  cfg = config.programs.neovim.catppuccin;
+  cfg = config.catppuccin.neovim;
   enable = cfg.enable && config.programs.neovim.enable;
 in
 {
-  options.programs.neovim.catppuccin = lib.ctp.mkCatppuccinOpt { name = "neovim"; };
+  options.catppuccin.neovim = lib.ctp.mkCatppuccinOpt { name = "neovim"; };
```

Comment:
Should we rename this to `nvim`? That's what the port repo is called after all 🤔